src/khoj/interface/web/config.html

```diff
@@ -1,70 +1,114 @@
+                        Setup
+                        <svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><path d="M5 12h14M12 5l7 7-7 7"></path></svg>
+                    </button>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="section">
+        <h2 class="section-title">Features</h2>
+        <div class="section-cards">
+            <div class="card">
+                <div class="card-title-row">
+                    <img class="card-icon" src="/static/assets/icons/chat.svg" alt="Chat">
+                    <h3 class="card-title">Chat</h3>
+                </div>
+                <div class="card-description-row">
+                <p class="card-description">Setup Khoj Chat</p>
+                </div>
+                <div class="card-action-row">
+                <button class="card-button" onclick="window.location.href='/config/processor/conversation';">Setup <svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><path d="M5 12h14M12 5l7 7-7 7"></path></svg></button>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="section">
+        <button id="save" type="submit">⚙️ Configure</button>
+    </div>
 </div>
-<h2>Processors</h2>
-<button onclick="window.location.href='/config/processor/conversation/';">
-    Conversation
-</button>
-<h1>Finalize</h1>
-<button id="regenerate" type="submit">Regenerate</button>
-<style>
-    body.khoj-configure {
-        padding: 0 10%
-    }
-    div#content-configuration, div#actions {
-        display: grid;
-        grid-template-columns: 1fr 1fr;
-        grid-gap: 12px;
-    }
-    button#regenerate {
-        background-color: #4CAF50;
-        border: none;
-        color: white;
-        padding: 15px 32px;
-        text-align: center;
-        text-decoration: none;
-        display: inline-block;
-        font-size: 16px;
-    }
-</style>
 <script>
-    var regenerate = document.getElementById("regenerate");
+    var regenerate = document.getElementById("save");
     regenerate.addEventListener("click", function(event) {
         event.preventDefault();
         regenerate.disabled = true;
-        regenerate.innerHTML = "Regenerating...";
+        regenerate.innerHTML = "Saving...";
         fetch('/api/update?force=true&client=web', {
             method: 'GET',
         })
         .then(response => response.json())
         .then(data => {
             console.log('Success:', data);
-            alert("Regenerated!");
+            alert("Saved!");
             regenerate.disabled = false;
-            regenerate.innerHTML = "Regenerate";
+            regenerate.innerHTML = "Save";
```

Comment:
What if this was `Refresh` instead? I think `Save` doesn't do justice to indicating to the user that the freshly retrieved data is now available via Khoj, but I also understand that `Regenerate` is overly technical.

Reply:
Changed it to "Configure"? I'm actually using "Configure" for the button default text. So these should anyway have read as Configure, Configuring, Configured. Let me know if that doesn't look fine